xml/containers-get.xml

```diff
@@ -0,0 +1,586 @@
+    strategy will also be applied to future versions of &slea;, e.g.
+    <literal>suse/sle16</literal> and so on. For &slea; 12 however, there is a
+    separate repository name for each service pack, e.g.
+    <literal>suse/sle12sp3</literal>, <literal>suse/sle12sp4</literal>,
+    <literal>suse/sle12sp5</literal>.
+   </para>
+  </sect2>
+
+  <sect2 xml:id="id-labels">
+   <title>Labels</title>
+   <para>
+    Labels are another way to identify images. All &slea; container image
+    labels begin with
+    <literal>com.suse.&lt;product&gt;.&lt;containername&gt;</literal> after
+    which follows a further specification. {opensuse} container images are
+    identified by a label that begins with
+    <literal>org.opencontainers</literal>.
+   </para>
+   <important>
+    <title>The <literal>com.suse.sle.base</literal> label is protected.</title>
+    <para>
+     With every new layer added to a base image, labels defined in the base
+     container get overwritten by labels in the derived image. And because the
+     base images are only the bare OS and any functionality added by the user
+     leads to the creation of a new layer, the labels get overwritten very
+     easily. In order to protect information about the origin of the image, the
+     label <literal>com.suse.sle.base</literal> has been added to &slea;
+     container images. This signifies that &suse; is the vendor of this image
+     and this label <emphasis role="strong">must not be overwritten in derived
+     images</emphasis>. For example the label
+     <literal>com.suse.sle.base.title</literal> contains the image name "SUSE
+     Linux Enterprise 15 Base Container", which is information that cannot be
+     overwritten with any other layer of the image.
+    </para>
+    <para>
+     The opposite is true for {opensuse} base images, where labels do get
+     overwritten. For example <literal>org.opencontainers.image.title</literal>
+     can get overwritten and will always contain the title of the latest added
+     layer.
+    </para>
+   </important>
+   <para>
+    A list of all currently defined labels follows:
+   </para>
+   <table frame="all" rowsep="1" colsep="1">
+    <title>Currently Defined Labels</title>
+    <tgroup cols="5">
+     <colspec colname="col_1" colwidth="20*"/>
+     <colspec colname="col_2" colwidth="20*"/>
+     <colspec colname="col_3" colwidth="20*"/>
+     <colspec colname="col_4" colwidth="20*"/>
+     <colspec colname="col_5" colwidth="20*"/>
+     <thead>
+      <row>
+       <entry align="left" valign="top">Must be provided by derived images</entry>
+       <entry align="left" valign="top">Labels</entry>
+       <entry align="left" valign="top">OCI Annotation</entry>
+       <entry align="left" valign="top">Description</entry>
+       <entry align="left" valign="top">Example</entry>
+      </row>
+     </thead>
+     <tbody>
+      <row>
+       <entry align="left" valign="top">
+        <para>
+         ✓
```

Comment:
An ASCII version of this would be desirable -- not sure this even comes out correctly in PDF. E.g. just use  "yes"/"no". (Same for the rest of the table)